packages/grafana-data/src/text/markdown.ts

```diff
@@ -21,7 +23,15 @@ export function renderMarkdown(str?: string, options?: RenderMarkdownOptions): s
     hasInitialized = true;
   }
 
-  const html = marked(str || '');
+  let opts = undefined;
+  if (options?.breaks) {
+    opts = {
+      ...markdownOptions,
+      breaks: true,
```

Comment:
ideally we could just set `breaks: true` for everything -- but that may cause issues with existing text panels!